controllers/helmchart_controller_test.go

```diff
@@ -694,6 +694,34 @@ var _ = Describe("HelmChartReconciler", func() {
 					storage.ArtifactExist(*got.Status.Artifact)
 			}, timeout, interval).Should(BeTrue())
 
+			When("Setting valid valuesFiles attribute", func() {
```

Comment:
I think it would make sense to add a check for the generated `values.yaml` file in the resulting helm chart, especially considering that we're merging the values, this would ensure the expected behavior.

Reply:
Yes, agreed. That would also protect against the 0-byte issue/bug we have had before.